config/routes.rb

```diff
@@ -7,7 +7,7 @@
   ActiveAdmin.routes(self)
 
   get 'statutory_instruments', to: 'statutory_instruments#index', as: 'statutory_instruments'
-  get 'statutory_instrument', to: 'statutory_instruments#show', as: 'statutory_instrument'
+  get 'statutory_instrument/:id', to: 'statutory_instruments#show', as: 'statutory_instrument'
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.